deploy/role.yaml

```diff
@@ -97,3 +97,13 @@ rules:
   - create
   - update
   - patch
+- apiGroups:
+    - integreatly.org
+  resources:
+    - grafanadashboards
+  verbs:
+    - get
+    - list
+    - watch
+    - create
+    - delete
```

Comment:
Do we really need a `delete` rights?

Reply:
though I don't know how to get the old namespace for deleting

Reply:
Oh, it could be a global var or prefer struct for storing previous collected value and if it changed, old must be deleted and new one can be created

Reply:
it needs some though, we need something that survives restart or upgrade

Reply:
oh, it can be populated at startup...